surfstat/surfstat_wrap.py

```diff
@@ -7,10 +7,10 @@
 import math
 import sys
 
-def matlab_init_surfstat():
+def matlab_init_surfstat(surfstat_path='./matlab'):
```

Comment:
Aha, I've seen your intention using a user-defined matlab path here.
However, no test function needs this change. They all work fine with matlab_init_surfstat() line. So, I'd not change it, it can stay fixed as earlier.

Reply:
The conflict arises because I changed the path but someone else changed it to output the engine. Whilst its indeed not strictly necessary for the tests - I've used matlab_init_surfstat in my own notebooks and these don't always have BrainStat/surfstat as their base directory.

Reply:
Have resolved the conflict. The behavior without input arguments is exactly identical to before i.e. no tests should be affected, but matlab_init_surfstat (and its associated functions) can now also be used without enforcing a specific working directory.

Reply:
cool.